teerex/src/tests/test_cases.rs

```diff
@@ -183,14 +182,8 @@ fn skip_attestation_add_sovereign_enclave_works_if_allowed() {
 			SgxAttestationMethod::Skip { proxied: false }
 		));
 		assert!(<SovereignEnclaves<Test>>::contains_key(&alice));
-		if let MultiEnclave::Sgx(sgx_enclave) = Teerex::sovereign_enclaves(&alice).unwrap() {
-			assert_eq!(
-				sgx_enclave.attestation_method,
-				SgxAttestationMethod::Skip { proxied: false }
-			);
-		} else {
-			panic!("wrong enclave type")
-		}
+		let MultiEnclave::Sgx(sgx_enclave) = Teerex::sovereign_enclaves(&alice).unwrap();
+		assert_eq!(sgx_enclave.attestation_method, SgxAttestationMethod::Skip { proxied: false });
```

Comment:
This is actually better than the previous code, as it won't compile anymore if we add more enclave types